[2022-06-27 13:31:05,668] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag1.model_B scheduled__2022-02-03T00:00:00+00:00 [queued]>
[2022-06-27 13:31:05,693] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag1.model_B scheduled__2022-02-03T00:00:00+00:00 [queued]>
[2022-06-27 13:31:05,697] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-06-27 13:31:05,698] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-06-27 13:31:05,699] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-06-27 13:31:05,739] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): model_B> on 2022-02-03 00:00:00+00:00
[2022-06-27 13:31:05,744] {standard_task_runner.py:52} INFO - Started process 57006 to run task
[2022-06-27 13:31:05,756] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'dag1', 'model_B', 'scheduled__2022-02-03T00:00:00+00:00', '--job-id', '181', '--raw', '--subdir', 'DAGS_FOLDER/dag1.py', '--cfg-path', '/tmp/tmp28hcies0', '--error-file', '/tmp/tmp__kvjfzr']
[2022-06-27 13:31:05,758] {standard_task_runner.py:80} INFO - Job 181: Subtask model_B
[2022-06-27 13:31:05,886] {task_command.py:370} INFO - Running <TaskInstance: dag1.model_B scheduled__2022-02-03T00:00:00+00:00 [running]> on host 43f8d36cfb0f
[2022-06-27 13:31:06,127] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=dag1
AIRFLOW_CTX_TASK_ID=model_B
AIRFLOW_CTX_EXECUTION_DATE=2022-02-03T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-02-03T00:00:00+00:00
[2022-06-27 13:31:06,129] {python.py:173} INFO - Done. Returned value was: 5
[2022-06-27 13:31:06,247] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag1, task_id=model_B, execution_date=20220203T000000, start_date=20220627T133105, end_date=20220627T133106
[2022-06-27 13:31:06,310] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-27 13:31:06,435] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
